amida-tech/indaba-client
# [INBA-903] Styling, check, error handling, reducer changes and new

#### What does this PR do?
Changes the Create Project Modal:
- Removes the summary field
- Changes title
- Adds instructions
- Some styling to prevent modal bounciness
- Adds an error explanation if a project already exists 

#### Related JIRA tickets:
INBA-903

#### How should this be manually tested?
Login as an admin. Go to the `Create` button and choose to create a new project. Confirm that the modal appears with the title fields, the new title and the instructions (see screenshot).

Try entering a Project that already exists in the system. The survey name does not matter. Click to submit and be stopped by the red error message.

Cancel to return your Projects homepage. Click on `Create` and and choose to create a new project. Confirm that the error _does not appear_ as the state has been reset. Enter a project name already in use as well as a survey name and confirm the error reappears.

Choose a different, unique project name and submit it. Confirm that it is accepted. 
#### Background/Context

#### Screenshots (if appropriate):
![image](https://user-images.githubusercontent.com/4026454/49114073-45b66480-f265-11e8-9408-c97bf439a308.png)
